### DISCUSSION TITLE
Has anyone tried to install on pan cam V3 fw version 4.50.4.7252.

### DISCUSSION THREAD

**Discussion by github-srj** [gtxaspec/wz_mini_hacks#710] [Q&A] [answered]
Any help or thoughts would be greatly appreciated.
I'm also contemplating use of Wyze / v3-unlocker · GitLab available here "https://git.i386.io/wyze/v3-unlocker".
Thanks in advance.

**Comment by gtxaspec** [comment]
What are you trying to achieve? Root? RTSP?

**Comment by github-srj** [comment]
Thanks for the quick response, it was unexpected.
RSTP would be sufficient, however rooting the devices would be beneficial as well.
It seems that Wyze has taken down the older firmware images.

**Reply by gtxaspec** [reply]
The pan cam v3 soc has secure boot enabled, it verified the signature of the SPL binary.  At this moment, you would need a hardware flash programmer to gain root on this model

**Comment by github-srj** [comment]
I was aware of the secure boot.  Without destroying the device, access to the physical hardware of the pan cam is just about impossible.  
There is a YouTube video where one of these is (less than successfully) dis-assembled.  Which basically precludes use of a flash programmer.   That is why I mentioned the v3-unlocker that is purportedly installable via the SD card, replaces u-boot and disables the secure boot functionality.  I'd feel much more confident about that If a dump of the boot code was available for comparison.  I'd rather not brick it.
You're thoughts?

**Reply by gtxaspec** [reply, marked as answer]
secure-boot is enabled on the SoC, not just u-boot.  There is an encryption key burned into the SoC's one-time-programmable efuse storage, so even if you replace u-boot, the SPL will complain that u-boot fails signature verification, and if you replace the SPL, the device will refuse to boot, at all.

We have a full dump of everything.
[panv3uboot.zip](https://github.com/gtxaspec/wz_mini_hacks/files/13968512/panv3uboot.zip)
here is the SPL+Uboot mtd0 partition.

the unlocker only works on devices that don't have secure boot enabled on the chip itself.

**Comment by github-srj** [comment]
Thanks for the assistance.
It seems that there is no way around it, short of having an acceptable key to provide signed binaries.
Saw that one person had an exploit, but it has been patched.
Do you know if the pan cam looks for / executes the upgrade.sh file?  That might still provide sufficient tinkering possibilities.

**Reply by gtxaspec** [reply]
Threre are exploits for older firmwares, but none have been published for the pan v3.
https://github.com/blasty/unwyze

**Comment by github-srj** [comment]
That was the one I had seen.
